crates/examples/infra/mod.rs

```diff
@@ -570,10 +570,10 @@ pub trait RunDA<
         let mut total_latency = 0;
         let mut num_latency = 0;
 
-        debug!("Sleeping for {start_delay_seconds} seconds before starting hotshot!");
+        println!("Sleeping for {start_delay_seconds} seconds before starting hotshot!");
```

Comment:
Looks like this PR 2858 have similar changes. Overall I think removing logging for print statements will make parsing logs for us more difficult when not in datadog (which is a big part of the workflow). Perhaps we can just set datadog to take info logs in docker and then filtein the UI, or filter the executable as `RUST_LOG=warn,hotshot_orchestrator::config=info` per Salman's suggestion?

Reply:
Yes, this PR is covered by that one. I set all of these to `info` in that PR, and I'm using `RUST_LOG=warn,hotshot_orchestrator::config=info` locally.